src/coreclr/vm/callconvbuilder.cpp

```diff
@@ -176,6 +177,8 @@ namespace
             return CorInfoCallConvExtension::FastcallMemberFunction;
         case CorInfoCallConvExtension::Thiscall:
             return CorInfoCallConvExtension::Thiscall;
+        case CorInfoCallConvExtension::Swift:
+            return CorInfoCallConvExtension::Swift;
```

Comment:
I assume the JIT will currently fail to compile, right?

Reply:
I believe the JIT will fail, but I'll validate next week.

Reply:
If it fails as expected consider me signed off.

Reply:
I've added the checks to ensure that the JIT fails when encountering CallConvSwift signatures.